packages/localize/src/message-translation.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2017 Ericson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import { injectable } from "inversify";
+const Globalize = require("globalize");
+const localeRequire = require("../i18n/locale.json");
+
+@injectable()
+export class MessageTranslation {
+
+    constructor() {
+
```

Comment:
empty line

Reply:
I will use it when implementing the preferences